client/create_dev_data.py

```diff
@@ -1,19 +1,16 @@
 #!/usr/bin/env python3
-import json
 import os
 import sys
 
 from sqlalchemy.orm.exc import NoResultFound
 
 from securedrop_client import db
-from securedrop_client.config import Config
 
 sdc_home = sys.argv[1]
 session = db.make_session_maker(sdc_home)()
 db.Base.metadata.create_all(bind=session.get_bind())
 
-with open(os.path.join(sdc_home, Config.CONFIG_NAME), "w") as f:
-    f.write(json.dumps({"journalist_key_fingerprint": "65A1B5FF195B56353CC63DFFCC40EF1228271441"}))
+os.environ["SD_SUBMISSION_KEY_FPR"] = "65A1B5FF195B56353CC63DFFCC40EF1228271441"
```

Comment:
Is this needed by the script still? I couldn't really tell.

Reply:
This turns out to be the flip-side of <https://github.com/freedomofpress/securedrop-client/pull/1883#pullrequestreview-2072345797>, both done and documented in ea333535c1c15f2fcdef88ded37441258e67c60e.